test/unit/traversing.js

```diff
@@ -743,6 +743,37 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
```

Comment:
`assert.ok( true, "String" )` was just a poor man's `QUnit.skip` before it existed; it's not necessary here. That's what I was trying to communicate in my last comment. :)